spec/@esm/esms_util_number_to_string_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+describe "ESMs_util_number_toString", requires_connection: true do
+  include_context "connection"
+
+  subject(:formatted_string) { execute_sqf!("#{number} call ESMs_util_number_toString") }
```

Comment:
THE GAME HAS IT'S OWN SCRIPTING LANGUAGE????

Reply:
Yes, called [SQF](https://community.bistudio.com/wiki/SQF_syntax) and it is a PITA to deal with sometimes